js/popover.js

```diff
@@ -65,10 +65,16 @@
     var $e = this.$element
     var o  = this.options
 
-    return $e.attr('data-content')
+    var content = $e.attr('data-content')
       || (typeof o.content == 'function' ?
             o.content.call($e[0]) :
             o.content)
+
+    if (['[object Function]', '[object Object]', '[object String]'].indexOf(Object.prototype.toString.call(content)) === -1) {
```

Comment:
At the point that I'm type checking I get the jQuery object as [object Object]. That's the reason for me using deep checking here instead of typeOf